packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -202,15 +271,42 @@ class MenuContent extends React.Component {
 
       return newItem;
     });
+    const boundingFrame = this.props.boundingRef ? this.props.boundingRef() : undefined;
+    const isFullScreen = MenuUtils.isFullScreen(
+      this.props.isHeightBounded,
+      this.props.isWidthBounded,
+      boundingFrame,
+      this.props.contentWidth,
+    );
+    const isSubMenu = this.props.index > 0;
+    const contentClass = cx([
+      'content',
+      { submenu: isSubMenu },
+      { 'hidden-page': this.props.isHidden },
+      { fullscreen: isFullScreen },
+    ]);
 
-    const header = this.buildHeader();
+    let header;
+    if (isFullScreen || isSubMenu) {
```

Comment:
Which test example should we view to see a fullscreen menu?

Reply:
The bounded menu is the "fullscreen" menu.